sync/class.jetpack-sync-module-posts.php

```diff
@@ -169,9 +169,9 @@ function expand_wp_insert_post( $args ) {
 		$post         = $args[1];
 		$update       = $args[2];
 		$is_auto_save = isset( $args[3] ) ? $args[3] : false; //See https://github.com/Automattic/jetpack/issues/7372
-		$just_published = isset( $args[4] ) ? $args[4] : false; //Preventative in light of above issue
+		$previous_state = isset( $args[4] ) ? $args[4] : false; //Preventative in light of above issue
```

Comment:
Previously, $just_published was a boolean, which made sense because we were specifying whether $just_published was true or false. With your changes, I'm not sure it makes sense to continue to assign a boolean to $previous_state. It might make more sense to assign a constant representing the previous state to $previous_state instead.

Reply:
make sense.